configurate-gson/src/main/java/ninja/leaping/configurate/gson/GsonConfigurationLoader.java

```diff
@@ -22,6 +22,9 @@
 import com.google.gson.stream.JsonReader;
 import com.google.gson.stream.JsonToken;
 import com.google.gson.stream.JsonWriter;
+
```

Comment:
spacing is wrong - should be all together

Reply:
Unless I'm missing something, the rest of the project seems to format things this way already?

Reply:
Nope nevermind, my bad. Will fix

Reply:
Ok should be sorted now - I've changed my codestyle config and organised everything using the existing style